frontend/src/app/landing-page/landing-page.component.html

```diff
@@ -42,5 +43,48 @@ <h3>Wo möchten Sie einkaufen?</h3>
       Sie sind Ladenbesitzer? Hier geht's lang...
     </a>
   </section>
-
+  <section id="bottom">
```

Comment:
Bitte nicht überall `id` verwenden - speziell nicht, wenn es gar nicht gebraucht wird. Braucht man nur einen Marker, den man per CSS referenzieren will, tut's eine gute alte `class` viel besser.

Erläuterung: `id` sollte wirklich **einmalig** sein. In unserer schönen Angular-Welt könnte es ja auch immer vorkommen, dass ich z.B. in einem ngFor drin stecke - und schwupps hab ich x-mal die gleiche ID im DOM. Im Normalfall passiert nix Schlimmes, aber schön isses nicht.